src/Validator.php

```diff
@@ -14,6 +14,17 @@
 
 abstract class Validator
 {
+
+    const TYPE_BOOLEAN = "boolean";
+    const TYPE_INTEGER = "integer";
+    const TYPE_FLOAT = "double"; /* gettype() returns "double" for historical reasons */
+    const TYPE_STRING = "string";
+    const TYPE_ARRAY = "array";
+    const TYPE_OBJECT = "object";
+    const TYPE_RESOURCE = "resource";
+    const TYPE_RESOURCE_CLOSED = "resource (closed)";
```

Comment:
Happy to remove object, resource, and resource (closed) if there's no use for those data types.

Reply:
@kodumbeats  Where is TYPE_RESOURCE and TYPE_RESOURCE_CLOSED used ?

Reply:
I guess nowhere in Appwrite :)  I added them early and didn't remove. Will take them out.

Reply:
Resolved in abfad1a